samples/src/JustSaying.Sample.Middleware/Program.cs

```diff
@@ -0,0 +1,125 @@
+﻿using JustSaying.Messaging;
+using JustSaying.Messaging.Middleware;
+using JustSaying.Sample.Middleware;
+using JustSaying.Sample.Middleware.Extensions;
+using JustSaying.Sample.Middleware.Handlers;
+using JustSaying.Sample.Middleware.Messages;
+using JustSaying.Sample.Middleware.Middlewares;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Serilog;
+
+Console.Title = "Middleware Example";
```

Comment:
Nit: this should be set back to the existing value in a `finally` when the application terminates.

Reply:
It's probably not needed really so I'll drop it